Sources/SQLiteDriver.swift

```diff
@@ -2,7 +2,15 @@ import Fluent
 
 public class SQLiteDriver: Fluent.Driver {
 
-	let database = SQLite()
+	var database = SQLite()
+
+	public init() {
+		database.initWithPath("Database/main.sqlite")
+	}
+
+	public init(path: String){
+		database.initWithPath(path)
```

Comment:
That way this will work

```swift
import Vapor
import Fluent
import FluentSQLite

SQLiteDriver.databasePath = "/path/to/db/main.sqlite"

let app = Application()
app.get ...
app.get ...

app.providers.append(FluentSqlite.Provider)

app.start()
```